src/rp2350/boot_stage2/pad_checksum

```diff
@@ -44,7 +44,7 @@ odata = idata + bytes(args.pad - len(idata))
 with open(args.ofile, "w") as ofile:
     ofile.write("// Padded and checksummed version of: {}\n\n".format(args.ifile))
```

Comment:
Perhaps this "and checksummed" file-comment is also misleading, as there's a code-comment a few lines above saying "No CRC..." ?
Also, looks like the `--seed` argument is _entirely_ unused by the RP2350 version of this script? :thinking: (Ahhh, but perhaps it's needed for CLI-compatibility with the [RP2040 version](https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/boot_stage2/pad_checksum))